yield_plugin/config/parameters.yaml

```diff
@@ -12,11 +10,38 @@ on_route_vehicle_collision_horizon_in_s: 10.0
 # Units: m/s
 # Value type: Desired
 obstacle_zero_speed_threshold_in_ms: 0.25
+# Time horizon until collision to commit to last stopping trajectory
+# NOTE: Only applies to situations if the ego was expected to stop completely due to the collision
+# not just yield and follow with a lower speed
+# Units: s
+time_horizon_until_collision_to_commit_to_stop_in_s: 3.0
+# Number of consecutive times the vehicle detects that the obstacle is behind
```

Comment:
What does behind in this context?

Reply:
remove this configuration parameter as currently it is unreachable since when there is no collision/object is behind the method returns nullptr anyway and resets the consecutive counter prevent it from incrementing to the limit
https://github.com/usdot-fhwa-stol/carma-platform/blob/f4ec70ad51c2691eb66d595589c0ac2152a68186/yield_plugin/src/yield_plugin.cpp#L838